templates/zerver/api/delete-queue.md

```diff
@@ -28,7 +28,7 @@ curl -X "DELETE" {{ api_url }}/v1/events \
 
 <div data-language="python" markdown="1">
 
-```
+```python
 #!/usr/bin/env python
```

Comment:
Okay, so this isn't a huge deal for two reasons:

* First, these `md` files will be rendered on our `/api` page, so you are not really supposed to read these files on GitHub.
* Second, because of the `data-language` attribute in the `div` write above the code block, adding this is usually redundant because it makes no difference on how these files are rendered.

Having said that, I am okay with this change for consistency and being explicit is almost always a good thing! :)